src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -596,6 +638,7 @@ public void testMpathwaysCall(PrintWriter out) {
 
 	private boolean isAllowedApiRequest(HttpServletRequest request) {
 		M_log.debug("isAllowedApiRequest(): called");
+		HttpSession session= request.getSession(true);
```

Comment:
The missing space will show up in some other formatted commit.  Might as well put it in.